pkgs/development/tools/parsing/tree-sitter/grammar.nix

```diff
@@ -27,10 +29,12 @@ stdenv.mkDerivation {
       "${source}/${location}"
   ;
 
+  NIX_CFLAGS_COMPILE = lib.optionalString stdenv.isDarwin "-I${lib.getDev libcxx}/include/c++/v1";
   buildInputs = [ tree-sitter ];
 
   dontUnpack = true;
   configurePhase = ":";
+  buildFlags = [ "USE_LIBCXX=1" ];
```

Comment:
Does this also need to be darwin-only? I don't see how `libcxx` would get injected into the package in the non-darwin case.

Reply:
Good question .. I will test without it. I was following the lead of other packages

Reply:
It has been updated